.github/workflows/integrationTest.yml

```diff
@@ -595,71 +609,143 @@ jobs:
           retry_wait_seconds: 5
           command: cd terraform/ec2/linux && terraform destroy --auto-approve
 
-#  @TODO add back when we add back windows tests
-#  EC2WinIntegrationTest:
-#    needs: [BuildMSI, GenerateTestMatrix]
-#    name: 'EC2WinIntegrationTest'
-#    runs-on: ubuntu-latest
-#    strategy:
-#      fail-fast: false
-#      matrix:
-#        arrays: ${{ fromJson(needs.GenerateTestMatrix.outputs.ec2_windows_matrix) }}
-#    permissions:
-#      id-token: write
-#      contents: read
-#    steps:
-#      - uses: actions/checkout@v2
-#
-#      - name: Configure AWS Credentials
-#        uses: aws-actions/configure-aws-credentials@v1
-#        with:
-#          role-to-assume: ${{ env.TERRAFORM_AWS_ASSUME_ROLE }}
-#          aws-region: us-west-2
-#
-#      - name: Cache if success
-#        id: ec2-win-integration-test
-#        uses: actions/cache@v2
-#        with:
-#          path: go.mod
-#          key: ec2-win-integration-test-${{ github.sha }}-${{ matrix.arrays.os }}
-#
-#      - name: Echo OS
-#        run: echo run on ec2 instance os ${{ matrix.arrays.os }}
-#
-#      - name: Verify Terraform version
-#        run: terraform --version
-#
-#        # nick-fields/retry@v2 starts at base dir
-#      - name: Terraform apply
-#        if: steps.ec2-win-integration-test.outputs.cache-hit != 'true'
-#        uses: nick-fields/retry@v2
-#        with:
-#          max_attempts: 3
-#          timeout_minutes: 15
-#          retry_wait_seconds: 5
-#          command: |
-#            cd integration/terraform/ec2/win
-#            terraform init
-#            if terraform apply --auto-approve \
-#            -var="ssh_key_value=${PRIVATE_KEY}" -var="ssh_key_name=${KEY_NAME}" \
-#            -var="github_repo=${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}.git" \
-#            -var="cwa_github_sha=${GITHUB_SHA}" -var="ami=${{ matrix.arrays.ami }}" \
-#            -var="test_dir=${{ matrix.arrays.test_dir }}" \
-#            -var="s3_bucket=${S3_INTEGRATION_BUCKET}" ; then
-#              terraform destroy -auto-approve
-#            else
-#              terraform destroy -auto-approve && exit 1
-#            fi
-#
-#      #This is here just in case workflow cancel
-#      - name: Terraform destroy
-#        if: ${{ cancelled() && steps.ec2-win-integration-test.outputs.cache-hit != 'true' }}
-#        uses: nick-fields/retry@v2
-#        with:
-#          max_attempts: 3
-#          timeout_minutes: 8
-#          retry_wait_seconds: 5
-#          command: cd integration/terraform/ec2/win && terraform destroy --auto-approve -var="ami=${{ matrix.arrays.ami }}"
+ 
+  EC2WinIntegrationTest:
+    needs: [BuildMSI, GenerateTestMatrix]
```

Comment:
The Linux test has:
```
  EC2LinuxIntegrationTest:
    needs: [MakeBinary, StartLocalStack, GenerateTestMatrix]
```
Why don't we need local stack for windows testing?

Reply:
`LocalStack` is created for the purpose of creating a mock server to validate if CloudWatchAgent with custom CA Bundle (being injected in the `common-config.toml`) able to send the metrics. That's not required by Mac and Windows. Therefore, it does not need.